Library/OcCpuLib/OcCpuLib.c

```diff
@@ -1272,6 +1280,7 @@ OcCpuScanProcessor (
   UINT32                  CpuidEbx;
   UINT32                  CpuidEcx;
   UINT32                  CpuidEdx;
+  BOOLEAN                 KVM_EXPOSE;
```

Comment:
According to EDK II codestyle variables should be named in camel case. Please rename to `KvmExpose`.